objcgen/processedtypes.cs

```diff
@@ -245,6 +245,10 @@ public class ProcessedProperty: ProcessedMemberBase {
 		public ProcessedProperty (PropertyInfo property, Processor processor) : base (processor)
 		{
 			Property = property;
+			if (HasGetter)
+				GetMethod = new ProcessedMethod (Property.GetGetMethod (), Processor);
```

Comment:
hmm... this ends up calling `Get[G|S]etMethod` methods multiple times (in most case, at at least each time `Has[G|S]etter` is called). What I had in mind was more like:

```
var g = Property.GetGetMethod ();
if (g != null)
    GetMethod = new ProcessedMethod (g, Processor);
```

and 

```
public bool HasGetter => GetMethod != null;
```